reference-architecture/gce-ansible/playbooks/roles/validate-app/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Create the validation project
+  command: oc new-project validate
+
+- name: Create Hello world app
+  shell: oc new-app https://github.com/openshift/cakephp-ex.git --name=php
```

Comment:
It might be worth merging the AWS validate-app here. I ran into a situation where the params here no longer worked and will make it easier for us to stay in sync